### ISSUE TITLE
Support XPATH selectors

### ISSUE THREAD

**Issue by gregberge** [smooth-code/jest-puppeteer#132]
As suggested in #77, expect-puppeteer should support XPATH selectors.

The API proposed in #77 looks good. Every matchers that accept a selector should accept either a string or an object.

The selector object will have two properties: `type` and `value`.
If the selector is a string, it will be converted to a CSS object selector.

```js
expect(page).toClick('#myId');

// or

expect(page).toClick({value: '#myId', type: 'css'});

// or

expect(page).toClick(
  {value: 'text()[starts-with(., "myText")]]', type: 'xpath'}
);
```

**Comment by becky26** [comment]
is there any update on the support for XPATH selectors? thank you!

**Comment by neoziro** [comment]
Not for now, waiting for a PR.

**Comment by UziTech** [comment]
fixed by #321
